crates/engine/tree/src/tree/root.rs

```diff
@@ -637,7 +634,7 @@ where
                                 std::mem::take(&mut current_multiproof),
                             );
                         } else if all_proofs_received && no_pending && updates_finished {
-                            let total_time = first_update_time
+                            let total_time = last_update_time
```

Comment:
unsure if we want to do this, because we already record it in `time_from_last_update`

Reply:
ok sg, will update #12639